### DISCUSSION TITLE
Establish "Shopping cart" for participants

### DISCUSSION THREAD

**Discussion by kaennchenstruggle** [pretalx/pretalx#1275] [Ideas]
As as frequent participant of congresses where pretalx had been used for scheduling I always wondered why I had to add each talk individually to my calendar. Moreover schedule changes did not lead to updated items in my calendar.

So here is my proposal: a "shopping cart"

Participants can add talks/wokshops/etc. to their cart until they have examined the whole event. Once done they can choose
- incorporate the cart as combined .ical into their calendar
-- allowing for an always up-to-date calendar
- have a link to the cart + combined .ical sent by email
-- allowing for an always up-to-date calendar, notifications by email once a new program is published (or only when items in cart change their slot), editing the cart

**Comment by rixx** [comment]
I don't believe yet another talk selection mechanism is a great idea when we already have starred talks – and having an iCal URL for those is already an open issue, tracking at #1002 

It's a bit bigger than "just add ical link, dummy", because we have exactly zero user identifying infrastructure for attendees at the moment, and we'll need to have *some* form of that to get this running.

**Comment by kaennchenstruggle** [comment]
I was in fact referring to something where a visitor can select talks based on their preference and embed an ical into their preferred tool. Sure this would imply some pseudonymized database table containing their selected talks. To increase the level of privacy (so visitors don't have to provide their email address) an edit link could be added to the ical entries.

I agree that this might be some programming effort. But as a frequent participant in such event I would highly appreciate this.

Can we add it to the feature list? I my opinion it can be combined with the "starred talks" feature.

**Reply by rixx** [reply]
What do you mean by "add it to the feature list"? There is an open issue, as I've linked above. Implementation will have to wait for either somebody to pick up the issue and open a Pull Request, or for us (me) to get around to it in time, or for a sponsor to sponsor feature development to get it prioritised.
